oak-jcr/src/test/java/org/apache/jackrabbit/oak/jcr/NamePathTest.java

```diff
@@ -212,6 +218,47 @@ public void testSpaceInNames() throws RepositoryException {
         session.getNode(n.getPath());
     }
 
+    @Test
+    @Ignore("OAK-10544")
+    public void testPrefixRemapping() throws NamespaceException, RepositoryException {
+        Random r = new Random();
+        int i1 = r.nextInt();
+        int i2 = r.nextInt();
+        String prefix = "nstest" + i1;
```

Comment:
can we make the prefix really hardcoded here? Adding the random string here just complicates the error output.

Reply:
In Oak we probably can assume that we're working on an empty repo. But that's something we wouldn't be able to rely on in TCK tests.